src/elasticSearchService.test.ts

```diff
@@ -380,4 +389,132 @@ describe('typeSearch', () => {
+                        logicalOperator: 'AND' as const,
+                    },
+                ],
+            ],
+            [
+                'greater than or equal',
+                [
+                    {
+                        key: 'age',
+                        value: ['21'],
+                        comparisonOperator: '>=' as const,
+                        logicalOperator: 'AND' as const,
+                    },
+                ],
+            ],
+            [
+                'less than or equal',
+                [
+                    {
+                        key: 'age',
+                        value: ['21'],
+                        comparisonOperator: '<=' as const,
+                        logicalOperator: 'AND' as const,
+                    },
+                ],
+            ],
+            [
+                'AND combination',
+                [
+                    {
+                        key: 'someFieldThatTellsIfTheResourceIsActive',
+                        value: ['AVAILABLE', 'PENDING'],
+                        comparisonOperator: '==' as const,
+                        logicalOperator: 'AND' as const,
+                    },
+                ],
+            ],
+            [
+                'OR combination',
+                [
+                    {
+                        key: 'someFieldThatTellsIfTheResourceIsActive',
+                        value: ['AVAILABLE', 'PENDING'],
+                        comparisonOperator: '==' as const,
+                        logicalOperator: 'OR' as const,
+                    },
+                ],
+            ],
+        ]).test('- %s', async (scenario: string, searchFilters: SearchFilter[]) => {
+            const fakeSearchResult = {
+                body: {
+                    hits: {
+                        total: {
+                            value: 1,
+                            relation: 'eq',
+                        },
+                        max_score: 1,
+                        hits: [
+                            {
+                                _index: 'patient',
+                                _type: '_doc',
+                                _id: 'ab69afd3-39ed-42c3-9f77-8a718a247742_1',
+                                _score: 1,
+                                _source: {
+                                    vid: '1',
+                                    id: 'ab69afd3-39ed-42c3-9f77-8a718a247742',
+                                    resourceType: 'Patient',
+                                },
+                            },
+                        ],
+                    },
+                },
+            };
+            (ElasticSearch.search as jest.Mock).mockResolvedValue(fakeSearchResult);
+            const es = new ElasticSearchService(searchFilters);
+            await es.typeSearch({
+                resourceType: 'Patient',
+                baseUrl: 'https://base-url.com',
+                queryParams: [{}],
```

Comment:
```suggestion
                queryParams: {},
```
queryParams is an object.  That should get rid of the `0.*` in the snapshots

Reply:
Ah, yep. Copy/paste error on my part. Fixed.